src/plugins/unified_search/public/dataview_picker/sorting_popover.test.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import type { IStorageWrapper } from '@kbn/kibana-utils-plugin/public';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+import { StubBrowserStorage } from '@kbn/test-jest-helpers';
+import type { DataViewListItemEnhanced } from './dataview_list';
+import { SortingService } from './sorting_service';
+
+import { SortingPopover } from './sorting_popover';
+
+describe('SortingPopover', () => {
+  let storage: IStorageWrapper;
+  let sortingService: SortingService<DataViewListItemEnhanced>;
+
+  const openPopover = (wrapper: ReactWrapper) => {
+    wrapper.find(`[data-test-subj="openPopoverButton"]`).last().simulate('click');
+  };
+
+  beforeEach(() => {
+    storage = new Storage(new StubBrowserStorage());
+    sortingService = new SortingService<DataViewListItemEnhanced>(
+      {
+        alphabetically: (item) => item.name ?? item.title,
+      },
+      storage
+    );
+  });
+
+  it('should render EuiPopover', () => {
```

Comment:
Let's replace the snapshot test with testing that the popover is found (or remove it entirely)